igc_lib.py

```diff
@@ -71,7 +74,148 @@ def _rawtime_float_to_hms(timef):
 
     return hms((time/3600), (time%3600)/60, time%60)
 
-
+class Turnpoint:
+    """ single turnpoint in a task.
+    
+    Attributes: 
+        lat: latitude as a float in degrees DD.DD
```

Comment:
It means decimal degrees, but that is superfluous as we already state that it is a float in degrees. removed